src/utils/localizeDateHelper.ts

```diff
@@ -1,21 +1,34 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const tz = require('timezone')(require('timezone/zones'), require('timezone/en_US.js'), require('timezone/sv_SE.js'), require('timezone/fr_FR.js'), require('timezone/nb_NO.js'), require('timezone/de_DE'), require('timezone/es_ES'), require('timezone/ja_JP'), require('timezone/ko_KR'));
-
+/* eslint-disable @typescript-eslint/no-var-requires,global-require */
 // tz docs https://bigeasy.github.io/timezone/
+const tz = require('timezone')(
+  // supported timezones
+  // timezone is the quirks of the area time-wise (eg daylight savings) and timezone
+  require('timezone/Asia/Seoul'),
+  require('timezone/Asia/Tokyo'),
+  require('timezone/Europe/Madrid'),
+  require('timezone/Europe/Busingen'),
+  require('timezone/Europe/Paris'),
+  require('timezone/Europe/Stockholm'),
+  require('timezone/America/New_York'),
+
+  // supported locales
+  // locale is related to language and place (eg, Spanish and Mexico)
+  require('timezone/ko_KR'),
+  require('timezone/ja_JP'),
+  require('timezone/es_ES'),
+  require('timezone/sv_SE'),
+  require('timezone/en_US'),
+  require('timezone/de_DE'),
+  require('timezone/fr_FR'),
+);
+
 function localizeDateHelper(
   date: string,
   targetDateFormat: string,
   language: string,
   timeZone: string,
 ): string {
   let locale = null;
-  // locale is the language, like English has January
```

Comment:
Sort of like the comments, helps for the new dev.  Could they just be updated a bit here?

Reply:
updated how?